### ISSUE TITLE
Generate $PATH

### ISSUE THREAD

**Issue by ciaranmcnulty** [jakzal/toolbox#438]
In the `phpqa` docker build we have to set the path to include e.g. `/QualityAnalyzer/bin`

It'd be good to internalise this to a toolbox command that  echos the new path items so I can do

```shell
export PATH="$PATH:$(toolbox export-path)"
```

**Comment by jakzal** [comment]
Yes, I considered it in the past, but it seemed like a lot of effort considering potential benefits.

QualityAnalyzer is the last tool that has to be on the PATH. Given the project is pretty much dead, I think it's time to retire it from the image.

**Comment by jakzal** [comment]
QualityAnalyzer is long gone and all the other tools are in a single known location. I think we can close this, also considering no activity in 3 years.
